src/node_options.cc

```diff
@@ -748,6 +753,10 @@ PerProcessOptionsParser::PerProcessOptionsParser(
             kAllowedInEnvironment);
 #endif
 #endif
+  AddOption("--use-largepages",
```

Comment:
This is not behind a flag so wouldn't the `node --use-largepages` be ignored on unsupported platforms?
 I think it will be better to just add `#else return false;` to the `IsLargePagesEnabled` to just have printed that it is unsupported.

Reply:
Currently we are able to distinguish between large pages not being supported, and not being enabled. To retain that distinction, `IsLargePagesEnabled()` would have to return some enum value.

We have [some ongoing work](https://github.com/intel/iodlr/tree/master/large_page) that provides detailed status indications about the various failures. I'd like to modify that work to include a status `map_unsupported_platform`, and then bring the implementation into Node.js wholesale.

Would it be OK to leave that integration beyond the scope of this PR and land it in a future PR?

Reply:
Perfectly fine. Great work.